app/presenters/single_content_item_presenter.rb

```diff
@@ -127,6 +127,12 @@ def link_text(metric_name)
     I18n.t("metrics.#{metric_name}.title").try(:downcase)
   end
 
+  def csv_url(metric_name)
+    from = @date_range.from
+    to = @date_range.to
+    "/metrics#{base_path}?metric_name=#{metric_name}&from=#{from}&to=#{to}.csv"
```

Comment:
Perhaps just remove the `.csv` bit from the end here? I don't think it adds much.